deps/README.md

```diff
@@ -63,6 +63,10 @@ Hiredis
 
 Hiredis is used by Sentinel, `redis-cli` and `redis-benchmark`. Like Redis, uses the SDS string library, but not necessarily the same version. In order to avoid conflicts, this version has all SDS identifiers prefixed by `hi`.
 
+1. `git subtree pull --prefix deps/jemalloc https://github.com/redis/redis.git <version-tag> --squash`<br>
+This should hopefully merge the local changes into the new version.
+2. Conflicts will arise (due to our changes) you'll need to resolve them and commit.
+
```

Comment:
this text is inside the hiredis section, but the command refers to jemalloc.
copy paste error?

Reply:
+copypasta. Also @yossigo mentioned the need to squash differently, so a force on top coming, also I note the submodule pastes to redis instead of hiredis - fixing here as well.